src/theme/contract.css.ts

```diff
@@ -177,6 +177,7 @@ export const vars = createGlobalThemeContract(
       captionLarge: null,
       captionMedium: null,
       captionSmall: null,
+      inherit: null,
```

Comment:
**Question:** Do you need `inherit` in contract?

Reply:
Sure, but I think you can define `inherit` at the sprinkles level instead of the contract. This way, someone who decides to create a custom theme won't need to pass `inherit` to the contract. We already have such cases defined in https://github.com/saleor/macaw-ui/blob/canary/src/theme/sprinkles.css.ts#L116.